constructor/nsis/main.nsi.tmpl

```diff
@@ -317,8 +321,13 @@ FunctionEnd
                 Install for all users, but don't add to PATH env var:$\n\
                     > $EXEFILE /InstallationType=AllUsers$\n\
                 $\n\
+{%- if has_python %}
                 Install for just me, add to PATH and register as system Python:$\n\
                     > $EXEFILE /RegisterPython=1 /AddToPath=1$\n\
+{%- else %}
+                Install for just me and add to PATH:$\n\
+                    > $EXEFILE /AddToPath=1$\n\
+{%- endif %}
```

Comment:
Does this option depend on having Python or not? I'd say the are independent:

```suggestion
{%- endif %}
                Install for just me and add to PATH:$\n\
                    > $EXEFILE /AddToPath=1$\n\
```

Reply:
They are not mutually exclusive, I just didn't want the message to be overly long with examples. I was referring to the comment a few lines above:

```
        # There seems to be a limit to how many chars per ${Print} we can pass.
        # The message will get truncated silently, no errors.
        # That's why we split the help message in two calls.
```

Reply:
Done - splitting wasn't needed.